backend/Squirrel.ConsoleApp.BL/Extensions/MappingExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Squirrel.ConsoleApp.Models.DTO;
+using System.Reflection;
+
+namespace Squirrel.ConsoleApp.BL.Extensions
+{
+    public static class MappingExtensions
+    {
+        public static Column MapToStructureRow(IList<string> columnNames, IList<string> rowValues)
+        {
+            var row = new Column();
+            columnNames = columnNames.Select(x => x.ToLower()).ToList();
+            var columns = typeof(Column).GetProperties(BindingFlags.Public | BindingFlags.Instance).Select(p => p.Name).ToList();
+
+            foreach (var propertyName in columns)
+            {
+                var property = typeof(Column).GetProperty(propertyName);
+                if (property != null && columnNames.Contains(propertyName.ToLower()))
+                {
+                    var value = rowValues[columnNames.IndexOf(propertyName.ToLower())];
+                    if (property.PropertyType == typeof(string))
+                        property.SetValue(row, value);
+                    else if (property.PropertyType == typeof(int?))
+                        property.SetValue(row, Parsing.ParseNullableInt(value));
+                    else if (property.PropertyType == typeof(bool?))
+                        property.SetValue(row, Parsing.ParseNullableBool(value));
+                }
+            }
+            return row;
+        }
+
+        public static Constraint MapToChecksRow(IList<string> columnNames, IList<string> rowValues)
```

Comment:
Reason for this and next function? If we have first one

Reply:
done first and second as one Generic but `MapToDataRow` needs because it's dynamic (we don't know the incoming structure) and it's useful to map via `Dictionary<string, string>` and skip the first 3 columns (the same data)